<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="76055" started-at="2021-08-02T20:57:21 IST" finished-at="2021-08-02T20:58:37 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="76055" started-at="2021-08-02T20:57:21 IST" finished-at="2021-08-02T20:58:37 IST">
      <class name="com.qa.tests.LoginPageTest">
        <test-method status="PASS" signature="setExtent()[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="setExtent" is-config="true" duration-ms="193" started-at="2021-08-02T20:57:21 IST" finished-at="2021-08-02T20:57:21 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="setup" is-config="true" duration-ms="23428" started-at="2021-08-02T20:57:21 IST" finished-at="2021-08-02T20:57:45 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="verifyLoginHeader()[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="verifyLoginHeader" duration-ms="124" started-at="2021-08-02T20:57:45 IST" finished-at="2021-08-02T20:57:45 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [HOME] but found [Home]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [HOME] but found [Home]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.tests.LoginPageTest.verifyLoginHeader(LoginPageTest.java:80)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLoginHeader -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="tearDown" is-config="true" duration-ms="1396" started-at="2021-08-02T20:57:45 IST" finished-at="2021-08-02T20:57:46 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=verifyLoginHeader status=FAILURE method=LoginPageTest.verifyLoginHeader()[pri:0, instance:com.qa.tests.LoginPageTest@1060b431] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="setup" is-config="true" duration-ms="10830" started-at="2021-08-02T20:57:46 IST" finished-at="2021-08-02T20:57:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="LoginPageTitleTest()[pri:1, instance:com.qa.tests.LoginPageTest@1060b431]" name="LoginPageTitleTest" duration-ms="35" started-at="2021-08-02T20:57:57 IST" finished-at="2021-08-02T20:57:57 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginPageTitleTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="tearDown" is-config="true" duration-ms="810" started-at="2021-08-02T20:57:57 IST" finished-at="2021-08-02T20:57:58 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginPageTitleTest status=SUCCESS method=LoginPageTest.LoginPageTitleTest()[pri:1, instance:com.qa.tests.LoginPageTest@1060b431] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="setup" is-config="true" duration-ms="13915" started-at="2021-08-02T20:57:58 IST" finished-at="2021-08-02T20:58:12 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="FAIL" signature="LoginTest(java.lang.String,java.lang.String)[pri:2, instance:com.qa.tests.LoginPageTest@1060b431]" name="LoginTest" duration-ms="9096" started-at="2021-08-02T20:58:12 IST" data-provider="getLoginData" finished-at="2021-08-02T20:58:21 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[user4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[test4]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [CRMPRO] but found [Free CRM - CRM software for customer relationship management, sales, and support.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [CRMPRO] but found [Free CRM - CRM software for customer relationship management, sales, and support.]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at com.qa.tests.LoginPageTest.LoginTest(LoginPageTest.java:73)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="tearDown" is-config="true" duration-ms="1051" started-at="2021-08-02T20:58:21 IST" finished-at="2021-08-02T20:58:23 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginTest status=FAILURE method=LoginPageTest.LoginTest(java.lang.String,java.lang.String)[pri:2, instance:com.qa.tests.LoginPageTest@1060b431] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="setup()[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="setup" is-config="true" duration-ms="8689" started-at="2021-08-02T20:58:23 IST" finished-at="2021-08-02T20:58:31 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="LoginTest(java.lang.String,java.lang.String)[pri:2, instance:com.qa.tests.LoginPageTest@1060b431]" name="LoginTest" duration-ms="4544" started-at="2021-08-02T20:58:31 IST" data-provider="getLoginData" finished-at="2021-08-02T20:58:36 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[rohanw]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[John@028]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- LoginTest -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="tearDown" is-config="true" duration-ms="791" started-at="2021-08-02T20:58:36 IST" finished-at="2021-08-02T20:58:37 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=LoginTest status=SUCCESS method=LoginPageTest.LoginTest(java.lang.String,java.lang.String)[pri:2, instance:com.qa.tests.LoginPageTest@1060b431] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="closeExtent()[pri:0, instance:com.qa.tests.LoginPageTest@1060b431]" name="closeExtent" is-config="true" duration-ms="363" started-at="2021-08-02T20:58:37 IST" finished-at="2021-08-02T20:58:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeExtent -->
      </class> <!-- com.qa.tests.LoginPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
